src/screens/VisualTests/VisualTests.stories.tsx

```diff
@@ -210,6 +211,19 @@ export const NoBuild: Story = {
       res(ctx.data({ build: null } as AddonVisualTestsBuildQuery))
     ),
   },
+  render: ({ ...args }) => {
+    return (
+      <VisualTests
+        {...args}
+        updateBuildStatus={(fn) => args.updateBuildStatus(typeof fn === "function" ? fn({}) : fn)}
+      />
+    );
+  },
```

Comment:
Wouldn't it be simpler to just do 

```ts
const updateBuildStatusAction = action('updateBuildStatus');
args: {
   updateBuildStatus: jest.fn((fn) => updateBuildStatusAction(typeof fn...))
}
```

Reply:
I also don't really see why a render method is necessary. Even if it is, I'm not sure its worthwhile.

Reply:
OK, an approach similar enough to @tmeasday suggestion worked.